packages/embark/src/lib/modules/deployment/index.js

```diff
@@ -108,7 +108,7 @@ class DeployManager {
             });
           }
         ], (err) => {
-          self.logger.error(err);
+          err && self.logger.error(err);
```

Comment:
```
embark: > eslint bin/embark src/bin/ src/lib/
embark: /home/vsts/work/1/s/packages/embark/src/lib/modules/deployment/index.js
embark:   111:11  error  Expected an assignment or function call and instead saw an expression  no-unused-expressions
embark: ✖ 1 problem (1 error, 0 warnings)
```

^ As see in CI.  The linter isn't happy, maybe need to do:
```suggestion
          if (err) self.logger.error(err);
```